.github/actions/e2e/action.yaml

```
@@ -0,0 +1,87 @@
+#
+# Copyright (C) 2019-2024 vdaas.org vald team <vald@vdaas.org>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+name: "Deploy Vald and Run E2E test"
+description: "An action to deploy Vald for E2E testing"
+inputs:
+  client_type:
+    description: "Set client type. e.g) go, python, node, java"
+    required: true
+  image_tag:
+    description: "Set image tag to deploy Vald. e.g) nightly, vx.x.x"
+    required: true
+    default: "nightly"
+
+runs:
+  using: "composite"
+  steps:
+    - uses: vdaas/vald-client-ci/.github/actions/setup-language@main
+      with:
+        client_type: ${{ inputs.client_type }}
+    - name: Get dependencies version
+      id: get_deps_version
+      shell: bash
+      run: |
+        K3S_VERSION=$(curl -s  ${VALD_URL}/versions/K3S_VERSION)
+        HELM_VERSION=$(curl -s ${VALD_URL}/versions/HELM_VERSION)
+
+        echo "K3S_VERSION=${K3S_VERSION}"   >> $GITHUB_OUTPUT
+        echo "HELM_VERSION=${HELM_VERSION}" >> $GITHUB_OUTPUT
+
+        echo "K3S_VERSION=${K3S_VERSION}"
+        echo "HELM_VERSION=${HELM_VERSION}"
```

Is it impossible to use tee command or is it intentional separation?
```suggestion
        echo "K3S_VERSION=${K3S_VERSION}"   | tee -a $GITHUB_OUTPUT
        echo "HELM_VERSION=${HELM_VERSION}" | tee -a $GITHUB_OUTPUT
```

---

Thank you for your feedback 🙏 I fixed it in the following commit!
https://github.com/vdaas/vald-client-ci/pull/5/commits/1ecccdebba7d8e3a29f5825116df87b42af579d7